login/login.html

```diff
@@ -6,13 +6,61 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="icon" href="../favicon.png" type="image/gif" sizes="16x16" />
     <title>Login</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.7/css/all.css">
     <link rel="stylesheet" href="login.css">
    
 </head>
 <body>
     
-   <div class="container" id="container">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#">
+        <img class="nav-img" src="../music.png" alt="Musicophilia Logo" />
+        <span id="webTitle">&mu;<span style="color: white;font-size:23px">SICOPHILIA</span></span>
+      </a>
+      <button
+        class="navbar-toggler"
+        id="myToggleBtn"
+        type="button"
+        data-bs-toggle="collapse"
+        data-bs-target="#navbarSupportedContent"
+        aria-controls="navbarSupportedContent"
+        aria-expanded="false"
+        aria-label="Toggle navigation"
+      >
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div
+        class="collapse navbar-collapse"
+        id="navbarSupportedContent"
+        style="justify-content: flex-end"
+      >
+        <ul class="navbar-nav">
+          <!--  me-auto mb-2 mb-lg-0 -->
+          <li class="nav-item">
+            <a class="nav-link current" aria-current="page" href="../index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link current" href="./About/index.html">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link current" href="../Contact-Us/contact.html">Contact Us</a>
```

Comment:
Check this href link

Reply:
Ok I'll do

Reply:
Means it is not redirecting to that page right

Reply:
Yes